docs/tour/exports.md

```diff
@@ -96,5 +96,5 @@ for Stdout output.
 For example, this writes a D2 script of `x -> y` and outputs it to a file `example.svg`.
 
 ```shell
-echo "x -> y" | d2 - - > example.svg
+echo "x -> y" | d2 - example.svg
```

Comment:
this is specifically showing using `-` for stdout.

Reply:
that's true, but only because `-` is input.

`d2 input.d2` for example would write `input.svg`, but `d2 input.d2 -` would write to stdout.

the example just shows both for convenience, even though it is redundant.